lib/remoteproc/remoteproc_virtio.c

```diff
@@ -144,7 +143,7 @@ static unsigned char rproc_virtio_get_status(struct virtio_device *vdev)
 	return status;
 }
 
-#ifndef VIRTIO_DEVICE_ONLY
+#if VIRTIO_DRIVER_SUPPORT
```

Comment:
could you define a IS_ENABLE macro 

`#if IS_ENABLE(VIRTIO_DRIVER_SUPPORT)`

Reply:
Sure, what behavior did you have in mind? Should this `IS_ENABLED()` macro return false when the symbol is not defined, should that even happen? Do we want this so we can accept things like `y` and `n` values, otherwise the macro would be just this:

`#define IS_ENABLED(option) (option)`